pkg/cmd/server/api/v1/types_test.go

```diff
@@ -351,7 +353,8 @@ func TestMasterConfig(t *testing.T) {
 		AssetConfig: &internal.AssetConfig{
 			Extensions: []internal.AssetExtensionsConfig{{}},
 		},
-		DNSConfig: &internal.DNSConfig{},
+		DNSConfig:             &internal.DNSConfig{},
+		AdmissionPluginConfig: map[string]string{"pluginName": "configFilePath"}, // Must be specified explicitly becase it's omitempty
```

Comment:
I would rather see a list of name/config pairs, rather than a list of names, and a map of config files